config/settings.yml

```diff
@@ -23,6 +23,8 @@
         :values:
           - machine
     :ignore_terminated_instances: true
+  :s3:
+    :inventory_object_refresh: true
```

Comment:
hm you are right, for me it's 7500s vs 60s, with the rails fix applied. So something is really wrong in the old refresh